qmcpy/stopping_criterion/Fixed_multilevel.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# In[5]:
+
+
+from ._stopping_criterion import StoppingCriterion
+from ..accumulate_data import MLQMCData
+from ..discrete_distribution import DigitalNetB2,Lattice,Halton
+from ..discrete_distribution._discrete_distribution import LD
+from ..true_measure import Gaussian
+from ..integrand import MLCallOptions
+from ..util import MaxSamplesWarning, ParameterError, MaxLevelsWarning
+from numpy import *
+from scipy.stats import norm
+from time import time
+import warnings
+
+
+class CubQMCFML(StoppingCriterion):
+    
+    def __init__(self, integrand, abs_tol=.05, alpha=.01, rmse_tol=None, n_init=256., n_max=1e10, 
+        replications=32., levels=3):
+        """
+        Args:
+            integrand (Integrand): integrand with multi-level g method
+            abs_tol (float): absolute tolerance
+            alpha (float): uncertaintly level.
+                If rmse_tol not supplied, then rmse_tol = abs_tol/norm.ppf(1-alpha/2)
+            rmse_tol (float): root mean squared error
+                If supplied (not None), then absolute tolerance and alpha are ignored
+                in favor of the rmse tolerance
+            n_max (int): maximum number of samples
+            replications (int): number of replications on each level
+            levels (int): number of levels of refinement >= 2
+
+        """
+        self.parameters = ['rmse_tol','n_init','n_max','replications']
+        # initialization
+        if rmse_tol:
+            self.rmse_tol = float(rmse_tol)
+        else: # use absolute tolerance
+            self.rmse_tol =  float(abs_tol) / norm.ppf(1-alpha/2)
+        self.n_init = float(n_init)
+        self.n_max = float(n_max)
+        self.replications = float(replications)
+        self.levels = levels
+        # QMCPy Objs
+        self.integrand = integrand
+        self.true_measure = self.integrand.true_measure
+        self.discrete_distrib = self.integrand.discrete_distrib
+        # Verify Compliant Construction
+        allowed_levels = ['fixed-multi']
+        allowed_distribs = [LD]
+        allow_vectorized_integrals = False
+        super(CubQMCFML,self).__init__(allowed_levels, allowed_distribs, allow_vectorized_integrals)
+        
+    def integrate(self):
+        # Construct AccumulateData Object to House Integration Data
+        self.data = MLQMCData(self, self.integrand, self.true_measure, self.discrete_distrib,
```

Comment:
I'm not sure how this stopping criterion works. The `MLQMCData` object is for adaptive multi-level problems and the adaptive mechanisms are used below e.g. in `self.data._add_level()`. However, this seems to be written for fixed multi-level problems? 

Do you have a writeup or reference on how this works?

Reply:
Guillem doesn't remember about this. So should we just ignore this and not merge it?

Reply:
Sure, that works

Reply:
I added the following to `.gitignore`: `stopping_criterion/Fixed_multilevel.py`. Did it work?

Reply:
Deleted the file